packages/vuetify/src/components/Vuetify/goTo/index.ts

```diff
@@ -50,7 +50,7 @@ export default function goTo (_target: GoToTarget, _settings: Partial<GoToSettin
 
     container.scrollTop = Math.floor(startLocation + (targetLocation - startLocation) * ease(progress))
 
-    if (progress === 1 || container.clientHeight + container.scrollTop === container.scrollHeight) {
+    if (progress === 1 || window.innerHeight + container.scrollTop === container.scrollHeight) {
```

Comment:
does it still work if you use the container?

Reply:
With container (`container.clientHeight`) works fine in Chrome and FF on macOS but not iOS.
Safari is impacted on both macOS and iOS.

With proposed change (`window.innerHeight`) works fine on all 3 browsers in macOs and iOS.

Reply:
I mean `goto({ container: ..., ...})`

Reply:
...sorry
when you pass container via goTo then inside the`container.scrollTop` is always 0 (it is read only?)

Reply:
Just had another look and perhaps even better solution to this problem (covering both scroll down and up) will be to change following:

`if (progress === 1 || container.clientHeight + container.scrollTop === container.scrollHeight)`
into
`if (progress === 1 )` - tested and works well.

However not sure what is the purpose of `container.clientHeight + container.scrollTop === container.scrollHeight` ?

Reply:
It stops the animation when scroll goes beyond the maximum value (for example you have small container but you want to do `goTo(aBigNmber)`)

Reply:
oh then perhaps this maximum value should be checked and adjusted before `requestAnimationFrame` is called?

Reply:
Yea let's do that.

Reply:
do you want me to create another pr where maximum value should be checked and adjusted before `requestAnimationFrame` is called?